Alignment/OfflineValidation/src/Trend.cc

```diff
@@ -146,9 +147,12 @@ Trend::Trend (const char * name, const char * dir,
     if (JSON.count("CMSlabel"))
       CMS = Form("#scale[1.1]{#bf{CMS}} #it{%s}", JSON.get<string>("CMSlabel").data());
 
+    if(JSON.get_child("trends").count("TitleCanvas")) lumi = Form("#scale[0.8]{%s}", JSON.get_child("trends").get<string>("TitleCanvas").data());
+
     assert(ymin < ymax);
```

Comment:
you may want to use a cleaner error message

Reply:
Are you referring to the assert statement even though the line with "TitleCanvas" is highlighted?